package.json

```diff
@@ -49,6 +49,6 @@
     "protractor": "^5.4.1",
     "ts-node": "~7.0.1",
     "tslint": "~5.11.0",
-    "typescript": "^3.1.1"
+    "typescript": "^3.1.6"
```

Comment:
Let's keep it `^3.1.1`, as in some environments it throws error on `^3.1.6`.

Reply:
Done :)